lib/child_process.js

```diff
@@ -81,25 +82,12 @@ function normalizeExecArgs(command /*, options, callback*/) {
     callback = arguments[2];
   }
 
-  if (process.platform === 'win32') {
-    file = process.env.comspec || 'cmd.exe';
-    args = ['/s', '/c', '"' + command + '"'];
-    // Make a shallow copy before patching so we don't clobber the user's
-    // options object.
-    options = util._extend({}, options);
-    options.windowsVerbatimArguments = true;
-  } else {
-    file = '/bin/sh';
-    args = ['-c', command];
-  }
-
-  if (options && options.shell)
-    file = options.shell;
+  // Make a shallow copy so we don't clobber the user's options object
+  options = util._extend({}, options);
```

Comment:
I'd prefer to use `Object.assign()`. I was waiting to see how #4593 played out, but the reaction seems to be good, so I'll replace this.